react-with-graphql-tutorial.md

```diff
@@ -307,11 +310,11 @@ That's it for the GraphQL query. You have learned everything about structuring y
 
 {{% chapter_header "GraphQL Mutation with GitHub's GraphQL API" "graphql-mutation-github-api" %}}
 
-This section introduces the GraphQL mutation. It complements the GraphQL query, because it is used for writing data instead of reading it. The mutation shares the same principles as the query: it has fields and objects, arguments and variables, fragments and operation names and last but not least directives and nested objects for the returned result. When performing a mutation, you can specify the data as fields and objects that should be returned after a mutation. So let's see how mutations work by using your GraphiQL application. Before you start making your first mutation, beware that you are using live GitHub data here. For instance, if you follow a person on GitHub by making a mutation, you will follow this person for real. Which is fine in the end, because following one another should be encouraged to explore or collaborate on projects.
+This section introduces the GraphQL mutation. It complements the GraphQL query because it is used for writing data instead of reading it. The mutation shares the same principles as the query: it has fields and objects, arguments and variables, fragments and operation names, as well as directives and nested objects for the returned result. With mutations you can specify data as fields and objects that should be returned after it 'mutates' into something acceptable. Before you start making your first mutation, be aware that you are using live GitHub data, so if you follow a person on GitHub using your experimental mutation, you will follow this person for real. Fortunately this sort of behavior is encouraged on Github.
 
-In this section, you will star a repository on GitHub, the repository you have requested before with a query, by using a mutation available {{% a_blank "from GitHub's API" "https://developer.github.com/v4/mutation/addstar" %}}. You can find the `addStar` mutation when browsing the "Docs" sidebar in GraphiQL too. If you don't like to star the repository, you can unstar it afterward again. It's up to you. But I encourage you to keep the star. The repository which you are going to star is a project for teaching developers about the fundamentals of React.
+In this section, you will star a repository on GitHub, the same one you used a query to request before, using a mutation {{% a_blank "from GitHub's API" "https://developer.github.com/v4/mutation/addstar" %}}. You can find the `addStar` mutation in the "Docs" sidebar. The repository is a project for teaching developers about the fundamentals of React.
```

Comment:
"... mutation in the "Docs" sidebar." to "... mutation in the "Docs" sidebar too."

Reply:
"The repository is a project for teaching developers about the fundamentals of React." to "The repository is a project for teaching developers about the fundamentals of React, so I guess it's a good thing to star it."

Reply:
* Adding "too" at the end of that sentence would be incorrect grammar.  Since "too" means "in addition to the related thing I said before" you need something in the previous sentence that it adds to.  The previous sentence tells the reader what to expect from the next section, but it makes no mention of the Docs sidebar for there to be a relation where "too" makes sense.
* Changed the second one to "...so starring it should prove useful beyond this tutorial."  As a person who is selling the idea that you're an expert on a subject (which you are), you don't want to begin any sentence with "I guess", even if you don't mean it literally.  I think readers need you to tell them directly "do this because it matters" not "you might want to do this because it could possibly matter later".  If they've made it this far in your book, they already trust you to tell them what they should be doing.